src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -318,4 +318,43 @@ public static void UnknownUrl_Throws_ProtocolException()
             Assert.True(exception.Message.Contains(protocolExceptionUri), string.Format("Expected exception message to contain '{0}'", protocolExceptionUri));
         }
     }
+
+    [Fact]
+    [ActiveIssue(194)]
+    [OuterLoop]
+    public static void DuplexCallback_Throws_FaultException()
+    {
+        DuplexChannelFactory<IWcfDuplexTaskReturnService> factory = null;
+        Guid guid = Guid.NewGuid();
+
+        NetTcpBinding binding = new NetTcpBinding();
+        binding.Security.Mode = SecurityMode.None;
+
+        DuplexTaskReturnServiceCallback callbackService = new DuplexTaskReturnServiceCallback();
+        InstanceContext context = new InstanceContext(callbackService);
+
+        try
+        {
+            var exception = Assert.Throws<FaultException<FaultDetail>>(() =>
+            {
+                using (factory = new DuplexChannelFactory<IWcfDuplexTaskReturnService>(context, binding, new EndpointAddress(Endpoints.Tcp_NoSecurity_TaskReturn_Address)))
```

Comment:
ChannelFactory suffers the same `using` pattern problem as ClientBase - see [ChannelFactory.cs](https://github.com/dotnet/wcf/blob/a891cd38db0fdc2cbd6282a727a23884e7b42472/src/System.Private.ServiceModel/src/System/ServiceModel/ChannelFactory.cs#L171) - so best practice is to avoid using the `using` pattern, see #168.